Curtains/Curtains.Domain/Data/CurtainsContext.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+
+namespace Curtains.Domain.Models
```

Comment:
correct the namespace - <code>Curtains.Domain.Data</code>

Reply:
good catch